core/src/main/java/de/danoeh/antennapod/core/service/QuickSettingsTileService.java

```diff
@@ -0,0 +1,69 @@
+package de.danoeh.antennapod.core.service;
+
+import android.content.ComponentName;
+import android.content.Intent;
+import android.os.Build;
+import android.os.IBinder;
+import android.service.quicksettings.Tile;
+import android.service.quicksettings.TileService;
+import android.util.Log;
+import android.view.KeyEvent;
+
+import androidx.annotation.RequiresApi;
+
+import de.danoeh.antennapod.core.preferences.PlaybackPreferences;
+import de.danoeh.antennapod.core.receiver.MediaButtonReceiver;
+import de.danoeh.antennapod.core.service.playback.PlaybackService;
+
+@RequiresApi(api = Build.VERSION_CODES.N)
+public class QuickSettingsTileService extends TileService {
+    /**
+     * Logging tag
+     * Missing final 'e' because of max 23 characters for log tags
+     */
```

Comment:
The second half of the comment is outdated. The first half is self-explanatory. So I would remove this comment as well :)

Reply:
Done, but just FYI the "Logging tag" comment is copied from other places (e.g. it's also in PlaybackService.java)

Reply:
Yeah, some places in AntennaPod have a lot of redundant comments :D